src/index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="style.css" />
+
+    <title>Henok's Portfolio</title>
+  </head>
+  <body>
+    <!-- Header -->
+    <header class="header container">
+      <!-- Logo -->
+      <div class="header__logo">My Logo</div>
+      <!-- Navigation -->
+      <nav class="header__nav">Desktop Navigation</nav>
+
+      <!-- Hamburger Menu -->
+      <img
+        class="header__hamburger-menu"
+        src="assets/icons/hamburger.svg"
+        alt="Hamburger menu icon"
+      />
+    </header>
+    <main>
+      <!-- Hero -->
+      <section class="hero container">
+        <h1 class="hero__header">
+          <span>I’m Henok.</span><span>Glad to see you!</span>
+        </h1>
+        <p class="hero__body">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hesitate to contact
+          me.
+        </p>
+
+        <!-- Social Media Links -->
+        <div class="hero__social-media-links">
+          <h2 class="hero__social-media-links__title">Let&apos;s connect</h2>
+          <div class="hero__social-media-links__icons">
+            <!-- Twitter -->
+            <a href="https://twitter.com/henokkhm">
+              <img src="assets/icons/twitter.svg" alt="Twitter icon" />
+            </a>
+
+            <!-- LinkedIn -->
+            <a href="https://www.linkedin.com/in/henokkhm/">
+              <img src="assets/icons/linked-in.svg" alt="linkedIn icon" />
+            </a>
+
+            <!-- Medium -->
+            <a href="#">
+              <!-- TODO: Get link to Medium -->
+              <img src="assets/icons/medium.svg" alt="Medium icon" />
+            </a>
+
+            <!-- GitHub -->
+            <a href="https://github.com/henokkhm">
+              <img src="assets/icons/github.svg" alt="GitHub icon" />
+            </a>
```

Comment:
- [ ] You have done a very great job to ensure that the social icons are links connected to your personal social handles 👏. However, I suggest that we ensure that The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link `(a tag)` as you have already done. Apart from being semantically correct, the a tag will let you connect your social networks with the icons, using href ☺️.